VaticanBAV/borget/BAVborget24.xml

```diff
@@ -0,0 +1,573 @@
+                          <title type="incomplete" ref="LIT1873Mangad#Third" />
+                        </msItem>
+                      </msItem>
+
+                      <msItem xml:id="ms_i1.4">
+                        <incipit xml:lang="gez">በስመ፡ አብ፡ <gap reason="ellipsis" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran"/> በብርሃን፡ ስሙ፡ ለእግዚአብሔር፡ ኢይትፈለጥ፡ ሰማይ፡ ወምድር፡ ፯መላእክት፡ ብርሃናውያን፡ እለ፡ ይነብሩ፡ ታሕተ፡ መንጦላዕቱ፡ ቅዱስ፡</incipit>
+                        <note>Only two prayers have been included instead of the usual seven.</note>
+
+                        <msItem xml:id="ms_i1.4.1">
+                          <title type="incomplete" ref="LIT1758Lefafa#FourthChapter"/>
+                        </msItem>
+
+                        <msItem xml:id="ms_i1.4.2">
+                          <title type="complete" ref="LIT1758Lefafa#FifthChapter"/>
+                        </msItem>
+
+                      </msItem>
+
+                      <msItem xml:id="ms_i1.5">
+                        <title type="complete" ref="LIT1758Lefafa#SixthChapter"/>
+                      </msItem>
+
+                      <msItem xml:id="ms_i1.6">
+                        <title type="complete" ref="LIT1758Lefafa#ThirdChapter"/>
+                      </msItem>
+
+                      <msItem xml:id="ms_i1.7">
+                        <title type="incomplete" ref="LIT1873Mangad#Sixth"/>
+                      </msItem>
+
+                    </msItem>
+
+                  </msContents>
+
+                  <physDesc>
+                    <objectDesc form="Scroll">
+                      <supportDesc>
+                        <support>
+                          <material key="parchment"/>
+                        </support>
+                        <extent>
+                          <note>Consists of <measure unit="strips">3</measure> strips sewn together.</note>
+                          <dimensions type="outer" unit="mm">
+                            <width>72</width>
+                            <height>1700</height>
+                          </dimensions>
+                        </extent>
+                      </supportDesc>
+                      <layoutDesc>
+                        <layout columns="1" writtenLines="306"/>
+                      </layoutDesc>
+                    </objectDesc>
+
+                    <handDesc>
+                      <handNote xml:id="h1" script="Ethiopic">
+                        <seg type="rubrication">
+                          Incipits, the name of Mary
+                        </seg>
+                      </handNote>
+                      <handNote xml:id="h2" script="Ethiopic">
+                        <seg type="rubrication">
+                          The name of Mary
+                        </seg>
+                      </handNote>
+                      <handNote xml:id="h3" script="Ethiopic">
+                        <seg type="rubrication">
+                          Occasionally the name ኃይለ፡ ጊዮርጊስ፡
```

Comment:
It would be good to create a person ID for him.

Reply:
Do you mean with a record and everything?

Reply:
Yes, we do that with all persons mentioned in manuscripts. I know that in cases such as this where only the most basic information is available it might seem unnecessary, but in the end this will add up to a lot of data which can be queried etc.